agent/src/main/java/io/jenkins/plugins/remotingkafka/Options.java

```diff
@@ -0,0 +1,18 @@
+package io.jenkins.plugins.remotingkafka;
+
+import org.kohsuke.args4j.Option;
+
+public class Options {
+
+    @Option(name = "-name", usage = "Name of the slave")
```

Comment:
"agent". Please do not use "slave" in new UIs, the term has been renamed in Jenkins 2.0